pmlc/rt/prng.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Intel Corporation
+#include "mlir/ExecutionEngine/RunnerUtils.h"
+#include "pmlc/compiler/registry.h"
+#include "pmlc/util/logging.h"
+
+extern "C" void
+plaidml_rt_prng(size_t stateRank, UnrankedMemRefType<float> *state,
+                size_t resultRank, UnrankedMemRefType<float> *result,
+                size_t newStateRank, UnrankedMemRefType<float> *newState) {
+  if (resultRank <= 0) {
```

Comment:
Since you're using a `size_t`, `<= 0` should never be possible.

On another note, use `unsigned` rather than `size_t`.

Reply:
Resolved